test/vtop_example.sh

```diff
@@ -68,6 +68,10 @@ function checkPodStatusWithTimeout() {
     sleep 1
   done
   echo -e "ERROR: checkPodStatusWithTimeout timeout to find pod matching:\ngot:\n$out\nfor regex: $regex"
+  vttabletPod=$(kubectl get pods | grep -E "vttablet" | head -n 1 | awk '{print $1}')
```

Comment:
Any reason not to use kubectl flags to get what we want rather than grep, head, and awk? That would be better IMO.

Reply:
Probably just ignorance on our part. I copy pasted this from the vtop repo, and we didn't really know how to use kubectl enough to be able to do it without needing grep, head and awk! Let me give it a try again!

Reply:
Oh damn no, that was added purely for debugging CI. But its working now. Let me clean that up! Looked at the code and realized it doesn't make sense to have this code there!

Reply:
Fixed